templates/navigation/navigation.html

```diff
@@ -34,7 +34,7 @@
              href="#"
```

Comment:
I see that in navigation.yaml there's a cta_link for each category, correct me if I'm wrong but should that be used here in the href? Right now all the categories' hrefs are '#' by default

Reply:
Yes, this is fine for now. There `href` attribute only exists here to make it clickable with keyboard, at a later date, when we have a non-js navigation, it will link to there.